library/src/main/java/com/klinker/android/link_builder/Link.java

```diff
@@ -29,6 +29,7 @@
     private String appendedText;
     private Pattern pattern;
     private int textColor = 0;
+    private int textColorHighlight = 0;
```

Comment:
Let's change some of the verbiage here, to better express, specifically, what this is for.

I think that something like `textColorOfHighlightedLink` would be more descriptive. I wasn't really understanding the goal of your commit because I didn't quite understand how it was being used.